bower.json

```diff
@@ -1,18 +1,16 @@
 {
   "name": "ember-test-utils",
   "dependencies": {
-    "ember": "~2.8.0",
+    "ember": "^2.10.2",
     "ember-cli-shims": "0.1.0",
-    "ember-cli-test-loader": "0.2.2",
     "ember-inflector": "1.3.1",
     "ember-mocha": "~0.8.11",
     "Faker": "3.0.1",
-    "lodash": "~4.5.0",
     "pretender": "~0.12.0",
     "sinon-chai": "^2.8.0",
     "chai-jquery": "^2.0.1"
   },
   "resolutions": {
-    "ember": "~2.8.0"
+    "ember": "^2.10.2"
```

Comment:
It stills works with Ember 2.8 and I added a CI build against 2.8 specifically. The only way we'll ever make it to Ember 2.10 is to start ensuring our various addons work in 2.10. I've already ensured all of our addons in `ciena-blueplanet` work in Ember 2.10 and have started the process on `ember-frost-core`. My goal is to try to get everything working in Ember 2.10 without making any major changes, in an effort to make 2.10 viable in our apps using LTS.